packages/scheduled-payments-client/package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "name": "@cardstack/scheduled-payments-client",
+  "version": "0.0.0",
+  "private": true,
+  "description": "The cardstack scheduled payments front-end app",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache --ext .js,.ts",
+    "lint:js:fix": "eslint . --fix",
+    "start": "ember serve",
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.8.1",
+    "@embroider/compat": "^1.8.3",
+    "@embroider/core": "^1.8.3",
+    "@embroider/webpack": "^1.8.3",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "@glint/core": "^0.9.4",
+    "@types/ember-data": "^4.4.4",
+    "@types/ember-data__adapter": "^4.0.0",
+    "@types/ember-data__model": "^4.0.0",
+    "@types/ember-data__serializer": "^4.0.0",
+    "@types/ember-data__store": "^4.0.1",
+    "@types/ember-qunit": "^5.0.0",
+    "@types/ember-resolver": "^5.0.11",
+    "@types/ember__application": "^4.0.2",
+    "@types/ember__array": "^4.0.2",
+    "@types/ember__component": "^4.0.10",
+    "@types/ember__controller": "^4.0.1",
+    "@types/ember__debug": "^4.0.1",
+    "@types/ember__engine": "^4.0.2",
+    "@types/ember__error": "^4.0.0",
+    "@types/ember__object": "^4.0.4",
+    "@types/ember__polyfills": "^4.0.0",
+    "@types/ember__routing": "^4.0.10",
+    "@types/ember__runloop": "^4.0.1",
+    "@types/ember__service": "^4.0.0",
+    "@types/ember__string": "^3.0.9",
+    "@types/ember__template": "^4.0.0",
+    "@types/ember__test": "^4.0.0",
+    "@types/ember__test-helpers": "^2.8.1",
+    "@types/ember__utils": "^4.0.1",
+    "@types/htmlbars-inline-precompile": "^3.0.0",
+    "@types/qunit": "^2.19.3",
+    "@types/rsvp": "^4.0.4",
+    "babel-eslint": "^10.1.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^2.4.2",
+    "ember-cli": "~4.6.0",
+    "ember-cli-app-version": "^5.0.0",
+    "ember-cli-babel": "^7.26.11",
+    "ember-cli-dependency-checker": "^3.3.1",
+    "ember-cli-htmlbars": "^6.1.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-terser": "^4.0.2",
+    "ember-cli-typescript": "^5.1.1",
+    "ember-cli-typescript-blueprints": "^3.0.0",
+    "ember-data": "~4.4.0",
```

Comment:
I think we can remove this and its various `@types` packages, as we have not been using Ember Data.

Reply:
cool, ya, I noticed on the projects this was not set, but I was not sure if it was a pattern or not, I'll remove it